app/code/Magento/Bundle/Test/Mftf/ActionGroup/AdminSelectAndDeleteProductsActionGroup.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AdminSelectAndDeleteProductsActionGroup">
```

Comment:
I would suggest a little bit different name. Something like `AdminDeleteAllProductsFromGridActionGroup`. 
Also, what do you think about moving this action group to the Catalog module?

Reply:
**Done**